more/free-programming-cheatsheets.md

```diff
@@ -128,6 +128,7 @@
 
 ### HTML and CSS
 
+* [Accessibility CheatSheet](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/) - Learn the Web, Thomas J Bradley (HTML)
```

Comment:
How do you know that Tomas J. Bradley is a creator?

Reply:
@davorpa ,

Based on their github repository

<img width="778" alt="image" src="https://user-images.githubusercontent.com/14177973/194248530-7dcd896e-fe4b-40ab-8e85-f01df1ac6875.png">

Reply:
Let's @eshellman decide how to write this